helm_remote/Tiltfile

```diff
@@ -83,7 +85,7 @@ def helm_remote(chart, repo_url='', repo_name='', values=[], set=[], namespace='
 def install_crds(name, directory):
     name += '-crds'
 
-    files = str(local(r"grep --include '*.yaml' --include '*.yml' -rEil '\bkind\s*:\s+CustomResourceDefinition\s*$' %s | xargs -r grep -L '^{{-'" % directory)).strip()
+    files = str(local(r"grep --include '*.yaml' --include '*.yml' -rEil '\bkind\s*:\s+CustomResourceDefinition\s*$' %s | xargs grep -L '^{{-'" % directory)).strip()
```

Comment:
This reversion will fix mac, but break other scenarios. See [this comment](https://github.com/tilt-dev/tilt-extensions/pull/38#issuecomment-649702694)